skfda/exploratory/visualization/_ddplot.py

```diff
@@ -0,0 +1,131 @@
+"""DD-Plot Module.
+
+This module contains the necessary functions to construct the DD-Plot.
+To do this depth is calculated for the two chosen distributions, and then
+a scatter plot is created of this two variables.
+"""
+
+from typing import List, Optional, TypeVar, Union
+
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from ...exploratory.depth.multivariate import Depth
+from ._utils import _get_figure_and_axes, _set_figure_layout_for_fdata
+
+T = TypeVar('T')
+
+
+class DDPlot:
+    """
+    DDPlot visualization.
+
+    Plot the depth of our fdata elements in two
+    different distributions, one in each axis. It is useful to understand
+    how our data is more related with one subset of data / distribution
+    than another one.
+    Args:
+        fdata: functional data set that we want to examine.
+        dist1: functional data set that represents the first distribution that
+            we want to use to compute the depth (Depth X).
+        dist2: functional data set that represents the second distribution that
+            we want to use to compute the depth (Depth Y).
+        depth_method: method that will be used to compute the depths of the
+            data with respect to the distributions.
+    Attributes:
+        depth_dist1: result of the calculation of the depth_method into our
+            first distribution (dist1).
+        depth_dist2: result of the calculation of the depth_method into our
+            second distribution (dist2).
+    """
+
+    def __init__(
```

Comment:
Maybe it would be useful to allow also `group` and `group_colors` parameters, to plot labeled data. We should discuss it.